docker-compose/_posts/2017-04-03-launching-your-first-environment.md

```diff
@@ -8,15 +8,31 @@ description: Using Docker Compose With Runnable
 ---
 
 
-Runnable makes it easy to launch containers from your compose file. 
+Runnable makes it easy to launch containers from your compose file.
 
-### Configuring your first repository
+---
+
+### Configuring Your First Repository
 
 
 Let's walkthrough configuring an example repository. The repository we will configure has the following Docker compose file:
-  ![Compose File]({{ site.baseurl }}/images/compose_config_composefile.png)
 
-From Compose file, it is clear that we expect to see 2 containers launch for this repository: a web container and a db container. To configure this repository on Runnable,
+    version: '2'
+    services:
+      db:
+        image: mongo
+        ports:
+          - "27017:27017"
+      web:
+        build: .
+        ports:
+          - "3000:3000"
+        links:
+          - "db"
+        environment:
+          -MONGODB_HOST=db
```

Comment:
You need a space after your hyphen on this line. Also you should delete the image that this section replaces.